src/main/steps/tasklistresponse/respond-to-allegations-of-harm/willing-to-respond/content.ts

```diff
@@ -0,0 +1,91 @@
+import { PartyType, YesOrNo } from '../../../../app/case/definition';
+import { TranslationFn } from '../../../../app/controller/GetController';
+import { FormContent } from '../../../../app/form/Form';
+import { isFieldFilledIn } from '../../../../app/form/validation';
+import { applyParms } from '../../../../steps/common/url-parser';
+import { DOWNLOAD_DOCUMENT_BY_TYPE, RESPOND_TO_APPLICATION } from '../../../../steps/urls';
+export * from './routeGuard';
```

Comment:
is it okay to keep * or shall we do import { routeGuard } from './routeGuard';

Reply:
* is fine as routeGuard can have multiple methods like post / get